spaceapi_server/src/server/mod.rs

```diff
@@ -58,7 +58,7 @@ impl SpaceapiServer {
 
         let router = self.route();
 
-        println!("Starting HTTP server on http://{}:{}...", host, port);
+        info!("Starting HTTP server on http://{}:{}...", host, port);
```

Comment:
I agree about the request logging, but leave this as a println. The reason is that the user should have at least some output to show that the program is working, even if he/she didn't configure the logger using the `RUST_LOG` env var.